src/main/java/org/jabref/gui/journals/AbbreviateAction.java

```diff
@@ -54,9 +62,17 @@ public void run() {
                 return false;
             };
 
-            boolean result = JabRefExecutorService.INSTANCE.executeAndWait(callable);
-            if (result) {
-                count++;
+            Future<Boolean> result = JabRefExecutorService.INSTANCE.executeAndReturn(callable);
+            futures.add(result);
+        }
+
+        for (Future<Boolean> future : futures) {
```

Comment:
Why not `ExecutorService.invokeAll()`? See https://stackoverflow.com/a/1250668/873282.

Reply:
Use `executeAll` from above?